core/extraction/towns_fund.py

```diff
@@ -562,7 +569,7 @@ def extract_risks(df_risk: pd.DataFrame, project_lookup: dict, programme_id: str
         "Proximity",
         "RiskOwnerRole",
     ]
-
+    # TODO: check this drop behaviour. Should we only drop if whole row is empty / null?
```

Comment:
Just checking these TODOs are to think about later rather than as part of this PR?

Reply:
Does it need to be captured in a ticket?

Reply:
Yeah, i'll leave them in for now, they are on me to check with Maeve and Enda

Reply:
I'll add tickets for them at some point